internal/rbd/clone.go

```diff
@@ -190,6 +190,12 @@ func (rv *rbdVolume) doSnapClone(ctx context.Context, parentVol *rbdVolume) erro
 	if err != nil {
 		return err
 	}
+	err = tempClone.unsetVolumeMetadata()
```

Comment:
the tempClone should contain the metadata same as the image it's created for.

Reply:
As discussed, let's do this on a need basis.